governance/polls/Ratification Poll - Update Data Insights Core Unit Mandate (MIP39c2-SP31) - April 11, 2022.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Ratification Poll for Update Data Insights Core Unit Mandate (MIP39c2-SP31) - April 11, 2022
+summary: This subproposal modifies the Data Insights Core Unit (“DIN”) mandate 
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp31-update-data-insights-core-unit-mandate/13790
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-25T16:00:00
+---
+# Ratification Poll for Update Data Insights Core Unit Mandate (MIP39c2-SP31) - April 11, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, April 11 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP39c2-SP31 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Data is important. The more data we have at our disposal, the better position we’ll be in to make good decisions and take advantage of new opportunities. Good data will also give us the justification and evidence we need to back up these decisions.
```

Comment:
"This subproposal modifies the Data Insights Core Unit (“DIN”) mandate previously ratified with [MIP39c2-SP22](https://forum.makerdao.com/t/mip39c2-sp22-adding-data-insights-core-unit-din-001/10208). It’s mostly about simplification of the previous document and stressing of our focus on L2 and cross-chain data as well as RWA datasets"

Also, fix format